targets/DeployDependencies.proj

```diff
@@ -78,10 +78,13 @@
   <UsingTask TaskName="PrereleaseResolveNuGetPackageAssets" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.Tasks.dll"/>
 
   <PropertyGroup>
+    <!-- netstandard is not a runnable framework moniker. If on .net core, target netcoreapp instead, which is runnable -->
```

Comment:
I might rephrase to

At build time, we target netstandard to be runnable on many runtimes. To get a set of runnable bits, we need to deploy a specific one, so we choose netcoreapp.

?